cap_client/cli/files_cli.py

```diff
@@ -63,17 +63,23 @@ def files():
     help="Uploads docker image."
 )
 @click.pass_context
-def upload(ctx, pid, file, yes, output_file=None, docker=False):
+def upload(ctx, pid, files, yes, output_file=None, docker=False):
     """Upload file to deposit with given pid."""
+    # disable file naming when uploading multiple files
+    output_file = None if len(files) > 1 else output_file
```

Comment:
if output_file parameter should be provided only if one file specified, we should inform the user, instead of just ignoring it
check [here](http://click.pocoo.org/5/options/#callbacks-for-validation) how to do this